cloud/reference/configuring-os-version.md

```diff
@@ -50,6 +50,7 @@ Flows will be run on iOS simulators&#x20;
 
```

Comment:
We should update the "Simulator Specs" above to indicate that the iPhone 11 default is configurble